### PR TITLE
Add Browser Compatibility

### PR DESCRIPTION
**Pull request by joncursi**
This replaces `request` with `isomorphic-unfetch` so that the same code can be run from a browser context.

https://github.com/developit/unfetch/tree/master/packages/isomorphic-unfetch

### CODE REVIEW

**Comment by joncursi** [comment]
The failing test is also failing on `master`, so this doesn't cause any new issues.

File: lib/common.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 'use strict';
 
-const request = require('request');
 const debug = require('debug')('app-store-scraper');
+const fetch = require('isomorphic-unfetch');
```
Comment (facundoolano) [inline lib/common.js]: I prefer to keep calling the var `request`

File: package.json [hunk 2]
```diff
@@ -25,6 +25,7 @@
   "homepage": "https://github.com/facundoolano/app-store-scraper#readme",
   "dependencies": {
     "debug": "^2.2.0",
+    "isomorphic-unfetch": "^2.0.0",
     "memoizee": "^0.3.10",
     "ramda": "^0.21.0",
     "request": "^2.79.0",
```
Comment (facundoolano) [inline package.json]: the request dependency isn't required anymore, right?

**Comment by facundoolano** [comment]
Fixed the tests in master, this is failing due to a style error.

**Comment by facundoolano** [comment]
Closing this PR, as it's outdated.

**Pull request closed without merging**